CMakeLists.txt

```diff
@@ -117,6 +117,8 @@ option(Tensile_NO_LAZY_LIBRARY_LOADING "Diasble loading kernels on demand?" OFF)
 # For roctx
 include(CMakeDependentOption)
 cmake_dependent_option(HIPBLASLT_ENABLE_MARKER "Enable roctx marker in hipBLASLt" ON "BUILD_SHARED_LIBS" OFF)
+# For rocRoller
+option(USE_ROCROLLER "Experimental: Build with the rocRoller library" OFF)
```

Comment:
CQE doesn't use install.sh, which means rocroller won't be built in 6.5 staging build even this PR get merged. Is this same as your expectation?

Reply:
Thanks @jichangjichang , I updated the PR because of this. Will need another approval.